### PR TITLE
Allow to share ciphers between Cozys

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

File: model/bitwarden/organization.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+const (
+	// OrgMemberInvited is used when the member is invited but has not yet
+	// accepted the invitation.
+	OrgMemberInvited = 0
+	// OrgMemberAccepted is used when the member is accepted but the owner has
+	// not yet confirmed that the fingerprint is OK.
+	OrgMemberAccepted = 1
+	// OrgMemberConfirmed is used when the member is confirmed, and has access
+	// to the organization key to decrypt/encrypt ciphers.
+	OrgMemberConfirmed = 2
```
Comment (Ldoppea) [inline model/bitwarden/organization.go:26]: What if the user reject the invitation? Is this a new status? Or is the member removed from the organisation?

File: web/bitwarden/organization.go [hunk 2]
```diff
@@ -1,64 +1,93 @@
...
-		Email:          string(email),
+		ID:             org.ID(),
+		Identifier:     nil, // Not supported by us
+		Name:           org.Name,
+		Key:            m.Key,
+		Email:          m.Email,
 		Plan:           "TeamsAnnually",
-		PlanType:       5, // TeamsAnnually plan
-		Seats:          2,
+		PlanType:       9,  // TeamsAnnually plan
```
Comment (Ldoppea) [inline web/bitwarden/organization.go:89]: This number may be subject to future changes. Shouldn't we recreate [this enum](https://github.com/bitwarden/jslib/blob/1016bbfb9eb28c220de8d2ab86d1f2757328f254/common/src/enums/planType.ts) in go or maybe use a constant?

File: model/bitwarden/organization.go (hunk 1 above)
Comment (nono) [reply model/bitwarden/organization.go:26]: I don't know yet, but my guess would be that the member is removed from the organization.

File: web/bitwarden/organization.go (hunk 2 above)
Comment (nono) [reply web/bitwarden/organization.go:89]: No, we don't really care. It's like `Seats`: the client is expecting a value, but it doesn't do anything from it. It is normally checked server-side.

**Comment by Ldoppea** [comment]
Commit https://github.com/cozy/cozy-stack/pull/3086/commits/a2339aa7f688c09339868a186e9bf0f4677490e7 breaks the notification service as it is guarded by the following code in `jslib/src/services/notifications.service.ts`:
```ts
private async processNotification(notification: NotificationResponse) {
  // ...

  const isAuthenticated = await this.userService.isAuthenticated();
  const payloadUserId = notification.payload.userId || notification.payload.UserId;
  const myUserId = await this.userService.getUserId();
  if (isAuthenticated && payloadUserId != null && payloadUserId !== myUserId) {
    return;
  }
```

With this commit, `payloadUserId !== myUserId` will always be `true`.

Side effect : created ciphers are not propagated between clients in realtime even if they are not in a shared folder.

**Comment by nono** [comment]
I had a git conflict. I made a rebase (from master) and a push-force to resolve it.

**Pull request merged**